delft/eris/status-page/status.js

```diff
@@ -1,7 +1,12 @@
-window.onload = (_) => {
```

Comment:
I've removed this wildcard `_` for the argument name because in JS all arguments are optional (with default to undefined) and arrity of functions is pretty much dynamic. While I personally also like this convention of denotating ignored arguments with `_` the problem i JS case is that one can never trust the number of arguments so it can be missleading when someone less experienced is reading the code and making assumptions about arity based on those wildcards even though in this case the arity won't change (unless some monkeypatching gets involved).

to make matters worse there are polupar libraries like [lodash](https://lodash.com/) and [underscore](https://underscorejs.org/) that make use of `_` as a namespace for functions and using `_` as argument name can cause some unfortunete hard to track shadowing of those namespaces.

Not a big deal just wanted but better safe than sorry I would say.